projects/golang/go/Makefile

```diff
@@ -69,8 +69,23 @@ prod-release: golang-build sync-artifacts-to-s3
 prod-release-images: fetch-eks-go-archive images
 
 .PHONY: fetch-eks-go-archive
-fetch-eks-go-archive:
-	curl -L -o $(VERSION_DIRECTORY)/archives/$(GOOS)/$(ARCH_LOWER)/$(GIT_TAG).$(GOOS)-$(ARCH_LOWER).tar.gz https://distro.eks.amazonaws.com/golang-$(GIT_TAG)/releases/$(BUILD_ID)/archives/$(GOOS)/$(ARCH_LOWER)/$(GIT_TAG).$(GOOS)-$(ARCH_LOWER).tar.gz --create-dirs
+fetch-eks-go-archive: download-go-archive validate-go-archive-checksum
+
+.PHONY: download-go-archive
+download-go-archive:
+	curl -L -o $(VERSION_DIRECTORY)/archives/linux/amd64/go1.19.5.linux-amd64.tar.gz https://distro.eks.amazonaws.com/golang-go1.19.5/releases/1/archives/linux/amd64/go1.19.5.linux-amd64.tar.gz --create-dirs
```

Comment:
Why did you remove the var use here, git_tag/build_id/arch/etc?

Reply:
@jaxesn not a final version, was testing locally without the vars.

Reply:
@jaxesn only 1.19 had the checksums added in S3 at that time.